recipe/session/session_test.go

```diff
@@ -83,6 +100,173 @@ func TestOutputHeadersAndSetCookieForCreateSessionIsFine(t *testing.T) {
 	assert.NotNil(t, cookieData["antiCsrf"])
 	assert.NotNil(t, cookieData["accessTokenExpiry"])
 	assert.NotNil(t, cookieData["refreshTokenExpiry"])
+
+	t.Run("verifySession returns 401 if multiple tokens are passed in the request", func(t *testing.T) {
+		req, err = http.NewRequest(http.MethodGet, testServer.URL+"/verifySession", nil)
+		assert.NoError(t, err)
+		req.Header.Add("Cookie", "sAccessToken="+cookieData["sAccessToken"])
+		req.Header.Add("Cookie", "sRefreshToken="+cookieData["sRefreshToken"])
+		req.Header.Add("Cookie", "sAccessToken="+cookieData["sAccessToken"])
+		req.Header.Add("Cookie", "sRefreshToken="+cookieData["sRefreshToken"])
+		res, err = http.DefaultClient.Do(req)
+		assert.NoError(t, err)
+		assert.Equal(t, http.StatusUnauthorized, res.StatusCode)
+
+		content, err := io.ReadAll(res.Body)
+		assert.NoError(t, err)
+		assert.Equal(t, `{"message":"try refresh token"}`, string(content))
+	})
+
+	t.Run("refresh endpoint throws a 500 if multiple tokens are passed and olderCookieDomain is undefined", func(t *testing.T) {
+		req, err = http.NewRequest(http.MethodPost, testServer.URL+"/auth/session/refresh", nil)
+		assert.NoError(t, err)
+		req.Header.Add("Cookie", "sAccessToken=accessToken1")
+		req.Header.Add("Cookie", "sAccessToken=accessToken2")
+		req.Header.Add("Cookie", "sRefreshToken=refreshToken1")
+		req.Header.Add("Cookie", "sRefreshToken=refreshToken2")
+		res, err = http.DefaultClient.Do(req)
+		assert.NoError(t, err)
+		cookieData = unittesting.ExtractInfoFromResponse(res)
+
+		assert.Equal(t, http.StatusInternalServerError, res.StatusCode)
```

Comment:
```suggestion
		assert.Equal(t, http.StatusInternalServerError, res.StatusCode)
		
		content, err := io.ReadAll(res.Body)
		assert.NoError(t, err)
		assert.Equal(t, "The request contains multiple session cookies. This may happen if you've changed the 'cookieDomain' value in your configuration. To clear tokens from the previous domain, set 'olderCookieDomain' in your config.\n", string(content))
```